mautic/mautic
# Add elFinder as filemanager for GrapesJs

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
Just added elFinder as filemanager to GrapesJs. Build based on customization documentation https://grapesjs.com/docs/modules/Assets.html#customization



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2.  Open the email
3. Insert image 
4. Check If elFinder works like in legacy  builder




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10902"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
